packages/@ourworldindata/grapher/src/barCharts/DiscreteBarChart.tsx

```diff
@@ -436,77 +436,81 @@ export class DiscreteBarChart
                         />
                     </React.Fragment>
                 )}
-                {series.map((series) => {
-                    // Todo: add a "placedSeries" getter to get the transformed series, then just loop over the placedSeries and render a bar for each
-                    const isNegative = series.value < 0
-                    const barX = isNegative
-                        ? yAxis.place(series.value)
-                        : yAxis.place(this.x0)
-                    const barWidth = isNegative
-                        ? yAxis.place(this.x0) - barX
-                        : yAxis.place(series.value) - barX
-                    const barColor = series.color
-                    const label = this.formatValue(series)
-                    const labelX = isNegative
-                        ? barX - label.width - labelToTextPadding
-                        : barX - labelToBarPadding
-
-                    // Using transforms for positioning to enable better (subpixel) transitions
-                    // Width transitions don't work well on iOS Safari – they get interrupted and
-                    // it appears very slow. Also be careful with negative bar charts.
-                    const result = (
-                        <g
-                            key={series.seriesName}
-                            className="bar"
-                            transform={`translate(0, ${yOffset})`}
-                        >
-                            <text
-                                x={0}
-                                y={0}
-                                transform={`translate(${labelX}, 0)`}
-                                fill="#555"
-                                dominantBaseline="middle"
-                                textAnchor="end"
-                                {...this.legendLabelStyle}
+                <g transform={`translate(0, ${barSpacing / 2})`}>
```

Comment:
I think it's better to just add `barSpacing / 2` to the initial `yOffset` used in `render()`.

Same goes for `StackedDiscreteBarChart`.